### DISCUSSION TITLE
How to use roles in Menu Items

### DISCUSSION THREAD

**Discussion by chrRtg** [balajidharma/basic-laravel-admin-panel#73] [Q&A]
While editing menu-items i can check the roles neccesary to see the menu item. But it seems the data does not get stored.

What I'm looking for is to show menu items according to the roles. In a best case a menu item (his route/url) is shown only, if the function behind the menu is accessibe for the current user.

Any idea on how to solve this?

**Comment by balajidharma** [comment]
The issue has been fixed. Do the composer update and try again.
